content/docs/intro/cloud-providers/azure/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Azure
-meta_desc: This page provides an overview of the Azure Provider for Pulumi.
+title: Azure Provider
```

Comment:
The word `provider` is redundant and none of others have it.

![](http://cnunciato-dropshare.s3.amazonaws.com/Screen-Recording-2020-02-25-14-57-20.gif)

```suggestion
title: Azure
```